graph.py

```diff
@@ -0,0 +1,77 @@
+
+class SimpleGraph(object):
+    '''This is a simple graph program that will allow us
+                to impliment a graph data structure'''
+    def __init__(self, dict_graph={}):
+        self.dict_graph = dict_graph
+
+    def node(self):
+        '''return a list of all nodes in the graph'''
+        return self.dict_graph.keys()
+
+    def edges(self):
+        '''return a list of all edges in the graph'''
+        edges = ''
+        for key, value in self.dict_graph.items():
+            edges = '{}->{}  {}'.format(key, value, edges)
+        return edges
```

Comment:
because you are passing back a string instead of tuples, the return value from this function will be useless to you later on when you are trying to use it for other purposes.  Never return representations of things instead of the things themselves, it restricts what you can do.